packages/client/src/components/Select/CustomSelect.jsx

```diff
@@ -1,63 +1,93 @@
 import React from 'react';
 import { FormControl, InputLabel, Select, MenuItem } from '@mui/material';
 import { useMovieList } from '../../context/movieListContext';
-import styled from '@emotion/styled';
+import FilterListIcon from '@mui/icons-material/FilterList';
 
 const CustomSelect = () => {
-  const { sortBy, onSortMovies, onChangeDirection } = useMovieList();
-
+  const { sortBy, onSortMovies, onChangeDirection, isClickedSame } =
```

Comment:
`isClickedSame` might be a bit misleading... Maybe a very explicit `isSameOptionClicked` would be better?

Reply:
Thanks for review, will fix it.